README.md

```diff
@@ -93,92 +96,96 @@ docker volume inspect pelion-edge-provisioner_mongo_data
 ```
 DEBUG=* pep --help
 ```
 
-
-If you are runnning pep cli on different machine than the pep-api-server docker container, then change PEP_SERVER_URL like this
+If you are runnning pep CLI on a different machine than the pep-api-server docker container, change PEP_SERVER_URL:
 
 ```
 PEP_SERVER_URL=http://<api-server-ip-address>:5151 pep --help
 ```
 
-
 ## Typical gateway provision flow
 
 1. Run factory-configurator-client-example on the gateway
 
-    Cross-compile latest version of [fcc-example](https://github.com/ARMmbed/factory-configurator-client-example) for your gateway platform. Instruction to compile fcc-example can be found here - `factory_configurator_utility.zip/docs/ft-demo/building-demo.html`
+    Cross-compile latest version of [fcc-example](https://github.com/ARMmbed/factory-configurator-client-example) for your gateway platform. You can find instructions to compile the fcc-example at `factory_configurator_utility.zip/docs/ft-demo/building-demo.html`.
 
-    Run -
+    Run:
+    
     ```
     $ ./factory-configurator-client-example.elf
     ```
 
-    To change the interface name -
+    To change the interface name:
+    
     ```
     ETHERNET_LINUX_IFACE_NAME=eth0 ./factory-configurator-client-example.elf
     ```
 
-    To change the entropy source -
+    To change the entropy source:
+    
     ```
     export ENTROPYSOURCE=<file-name>
     ```
 
-1. Install the `pep` cli on the gateway and run
+1. Install the `pep` CLI on the gateway, and run:
     1. `$ pep create-one-identity -s <serial_number> -w <hardware_version>`
         <img src="./assets/pep-create-one-identity.png" width="927" height="550"/>
 
     1. `$ pep get-one-identity -s <serial_number> -i <gateway_ip> -p <fcc_port>`
         <img src="./assets/pep-get-one-identity.png" width="927" height="464"/>
 
-1. factory-configurator-client-example creates a `pal` folder and get-one-identity creates `identity.json` file.
+1. factory-configurator-client-example creates a `pal` folder, and get-one-identity creates `identity.json` file.
 
     1. Place and rename the `pal` folder to the location specified by compile time flag `PAL_FS_MOUNT_POINT_PRIMARY` of mbed-edge.
     1. Place `identity.json` to the location specified by platform_readers/identity_path of maestro configuration file.
 
-
 ## Troubleshooting
 
-1. On gateway when compiling factory-configurator-client-example if you get error
+1. When compiling factory-configurator-client-example, if you see this error on the gateway:
 
     ```
     ImportError: No module named site
     ```
 
-    then try unsetting the python path
+    Try unsetting the Python path:
 
     ```
     unset PYTHONPATH
     unset PYTHONHOME
     ```
 
-1. If mbed-cli is not installed, follow these [instructions](https://os.mbed.com/docs/mbed-os/v5.9/tutorials/quick-start-offline.html)
+1. If Arm Mbed CLI is not installed, follow these [instructions](https://os.mbed.com/docs/mbed-os/latest/tutorials/quick-start-offline.html)
```

Comment:
We never mentioned Arm Mbed CLI needing to be installed before. Should that be a prepreq?